app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
- Hey :clap:, your show page throws error because it can't find the user with the id. Currently, since you just want to show a particular user, you can for now just remove the extra lines of code and update the show action to `def show: end` and this will fix the issue. Please, make the update.